### ISSUE TITLE
Option for sampling data onto points from mesh

### ISSUE THREAD

**Issue by MatthewFlamm** [MatthewFlamm/pyransame#16]
Currently to sample data (point or cell data) onto the sampled points, it is a manual procedure.  It would be useful to have something like

```py
pyransame.random_surface_points(mesh, n=500, sample_data=True)
```

**Comment by MatthewFlamm** [comment]
This would return a PyVista object (`PolyData`) rather than np.ndarray.  When this is implemented, if possible, we should have type overrides, but it may not be possible.

**Comment by MatthewFlamm** [comment]
Rather than a parameter, it could be a separate function that is mostly a wrapper.

**Comment by MatthewFlamm** [comment]
Implemented in #87.
